spec/21-http_response_header_format.md

```diff
@@ -34,12 +34,10 @@ The dash (`-`) character is used as a delimiter between fields.
 #### version
 
 ``` abnf
-version         = 2HEXDIGLC   ; this document assumes version 00. Version 255 is forbidden
+version         = 2HEXDIGLC   ; this document assumes version 00. Version ff is forbidden
 ```
 
-The value is US-ASCII encoded (which is UTF-8 compliant).
-
-Version (`version`) is 1 byte representing an 8-bit unsigned integer. Version `255` is invalid. The current specification assumes the `version` is set to `00`.
+Version (`version`) is an 8-bit unsigned integer value, serialized as an ASCII string with two characters. Version 255 (`"ff"`) is invalid. This specification assumes the version is set to 0 (`"00"`).
```

Comment:
Do we "assume" version is set to 00 or do we require it? We can keep the existing wording if it is easier but I think something like this is more accurate.

```suggestion
Version (`version`) is an 8-bit unsigned integer value, serialized as an ASCII string with two characters. Version 255 (`"ff"`) is invalid. This document specifies version 0 (`"00"`) of the traceparent header.
```

Reply:
Good point. This section is about the `traceresponse` header though. I'll fix that in a follow up commit.

Reply:
(For context, this is not yet the backport to level-2, I just ported the changes for traceparent also to the same fields for traceparent.)

Reply:
For the record, I commited this change and then rewrote the commit to say of the `traceresponse` header.